File: app/src/main/java/com/lifedawn/bestweather/retrofit/responses/kma/ultrasrtncstresponse/UltraSrtNcstRoot.java
package com.lifedawn.bestweather.retrofit.responses.kma.ultrasrtncstresponse;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.lifedawn.bestweather.retrofit.responses.kma.VilageFcstResponse;

public class UltraSrtNcstRoot {
	@Expose
	@SerializedName("response")
	private VilageFcstResponse response;
	
	
	public void setResponse(VilageFcstResponse response) {
		this.response = response;
	}
	
	public VilageFcstResponse getResponse() {
		return response;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/kma/json/ultrasrtncstresponse/UltraSrtNcstRoot.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.ultrasrtncstresponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.vilagefcstcommons.VilageFcstResponse

class UltraSrtNcstRoot {
    @Expose @SerializedName("response") var response: VilageFcstResponse? = null
}